src/code/z_player_lib.c

```diff
@@ -44,197 +45,224 @@ u8 gPlayerModelTypes[][5] = {
     gLinkChildLeftFistAndKokiriSwordNearDL,
-    0x06027078,
+    gLinkAdultLeftHandHoldingMasterSwordFarDL,
     gLinkChildLeftFistAndKokiriSwordFarDL,
 };
 
 Gfx* D_80125E38[] = {
-    0x06021F78,
+    gLinkAdultLeftHandHoldingMasterSwordNearDL,
     gLinkChildLeftFistAndKokiriSwordNearDL,
-    0x06027078,
+    gLinkAdultLeftHandHoldingMasterSwordFarDL,
     gLinkChildLeftFistAndKokiriSwordFarDL,
 };
 
-Gfx* D_80125E48[] = {
-    0x06022498,
+Gfx* sRightHandDLs[] = {
+    gLinkAdultRightHandNearDL,
     gLinkChildRightHandNearDL,
-    0x060274F8,
+    gLinkAdultRightHandFarDL,
     gLinkChildRightHandFarDL,
 };
 
-Gfx* D_80125E58[] = {
-    0x060226E0,
+Gfx* sRightHandClosed2DLs[] = {
+    gLinkAdultRightHandClosedNearDL,
     gLinkChildRightHandClosedNearDL,
-    0x06027690,
+    gLinkAdultRightHandClosedFarDL,
     gLinkChildRightHandClosedFarDL,
 };
 
-Gfx* D_80125E68[] = {
-    0x06022DA8,
-    gLinkChildRightHandHoldingFairySlingshotNearDL,
-    0x06027B88,
-    gLinkChildRightHandHoldingFairySlingshotFarDL,
+Gfx* sHandHoldingFirstPersonWeaponDLs[] = {
+    gLinkAdultRightHandHoldingBowNearDL,
+    gLinkChildRightHandHoldingSlingshotNearDL,
+    gLinkAdultRightHandHoldingBowFarDL,
+    gLinkChildRightHandHoldingSlingshotFarDL,
 };
 
-Gfx* D_80125E78[] = {
-    0x06023160,
+Gfx* sSwordAndSheathDLs[] = {
+    gLinkAdultMasterSwordAndSheathNearDL,
     gLinkChildSwordAndSheathNearDL,
-    0x06027F00,
+    gLinkAdultMasterSwordAndSheathFarDL,
     gLinkChildSwordAndSheathFarDL,
 };
 
-Gfx* D_80125E88[] = {
-    0x060249D8,
+Gfx* sSheathDLs[] = {
+    gLinkAdultSheathNearDL,
     gLinkChildSheathNearDL,
-    0x06028150,
+    gLinkAdultSheathFarDL,
     gLinkChildSheathFarDL,
 };
 
-Gfx* D_80125E98[] = {
-    0x06035330,
+Gfx* sWaistDLs[] = {
+    gLinkAdultWaistNearDL,
     gLinkChildWaistNearDL,
-    0x0602F530,
+    gLinkAdultWaistFarDL,
     gLinkChildWaistFarDL,
 };
 
 Gfx* D_80125EA8[] = {
-    0x06022DA8,
-    gLinkChildRightHandHoldingFairySlingshotNearDL,
-    0x06027B88,
-    gLinkChildRightHandHoldingFairySlingshotFarDL,
+    gLinkAdultRightHandHoldingBowNearDL,
+    gLinkChildRightHandHoldingSlingshotNearDL,
+    gLinkAdultRightHandHoldingBowFarDL,
+    gLinkChildRightHandHoldingSlingshotFarDL,
 };
 
-Gfx* D_80125EB8[] = {
-    0x06024698,
+Gfx* sFairyOcarinaDLs[] = {
+    gLinkAdultRightHandHoldingOotNearDL,
     gLinkChildRightHandHoldingFairyOcarinaNearDL,
-    0x06028F58,
+    gLinkAdultRightHandHoldingOotFarDL,
     gLinkChildRightHandHoldingFairyOcarinaFarDL,
 };
 
-Gfx* D_80125EC8[] = {
-    0x06024698,
+Gfx* sOotDLs[] = {
+    gLinkAdultRightHandHoldingOotNearDL,
     gLinkChildRightHandAndOOTNearDL,
-    0x06028F58,
+    gLinkAdultRightHandHoldingOotFarDL,
     gLinkChildRightHandHoldingOOTFarDL,
 };
 
-Gfx* D_80125ED8[] = {
-    0x06024D70,
+//! @bug The 'far' display list is never used
+Gfx* sHookshotDLs[] = {
+    gLinkAdultRightHandHoldingHookshotNearDL,
     gLinkChildRightHandNearDL,
```

Comment:
I wouldnt call this a bug, we have know way of really knowing. Its possible they ended up not wanting to use the far dlist in the end. You can remove the bug tag and say "a far dlists exists for holding hookshot, but it is not used"

Reply:
I feel that given the fact that the child DLs have near and far it was likely intended for adult to also use near and far.

Reply:
likely intended sure, but its not a bug.
It was likely, in my opinon, a choice on their part, to put the near dlist as far so it would be used for both on purpose. for all we know they decided the far one didnt look good.

Reply:
the comment i suggested with the removal of the bug tag is more appropriate imo

Reply:
possibly put it next to where the far dlist would go. at the top its not clear what exactly its talking about